v22.2/create-schedule-for-backup.md

```diff
@@ -90,10 +90,19 @@ If you encounter a bug, please [file an issue](file-an-issue.html).
 - We recommend that you schedule your backups at a cadence that your cluster can keep up with; for example, if a previous backup is still running when it is time to start the next one, adjust the schedule so the backups do not end up falling behind or update the [`on_previous_running` option](#on-previous-running-option).
 - To prevent scheduled backups from falling behind, first determine how long a single backup takes and use that as your starting point for the schedule's cadence.
 - Ensure you are monitoring your backup schedule (e.g., [Prometheus](monitor-cockroachdb-with-prometheus.html)) and alerting metrics that will confirm that your backups are completing, but also that they're not running more concurrently than you expect.
-- Ensure that your [GC window](configure-replication-zones.html#gc-ttlseconds) is long enough to accommodate your backup schedule, otherwise your incremental backups will throw [an error](common-errors.html#protected-ts-verification-error). For example, if you set up your schedule to be `RECURRING '@daily'` but your GC window is less than 1 day, all your incremental backups will fail.
 - The `AS OF SYSTEM TIME` clause cannot be set on scheduled backups. Scheduled backups are started shortly after the scheduled time has passed by an internal polling mechanism and are automatically run with `AS OF SYSTEM TIME` set to the time at which the backup was scheduled to run.
 - If you want to schedule a backup using temporary credentials, we recommend that you use `implicit` authentication; otherwise, you'll need to drop and then recreate schedules each time you need to update the credentials.
 
+### Protected timestamps and scheduled backups 
+
+{% include_cached new-in.html version="v22.2" %} Scheduled backups ensure that the data to be backed up is protected from garbage collection until it has been successfully backed up to a certain time. This active management of [protected timestamps](architecture/storage-layer.html#protected-timestamps) means that you can run scheduled backups at a cadence independent from the [GC TTL](configure-replication-zones.html#gc-ttlseconds) of the data. This is unlike non-scheduled backups that are tightly coupled to the GC TTL. See [Garbage collection and backups](take-full-and-incremental-backups.html#garbage-collection-and-backups) for more detail.
```

Comment:
Question: what does "to a certain time" mean here? The sentence looks perfectly-functional without this qualifier. Initially I assumed "up to the configured `gc.ttlseconds` value", i.e. 25 hours (by default) or 10m (by recommendation)), but: clarifying that this is _not_ the case is in fact the goal of this PR! So I'm not sure what we mean here. Line 100 goes on to state this wonderfully, maybe that's the leading sentence?

Reply:
"to a certain time" here would represent the last backup. That is, the scheduled backup would be backing up everything (depending on whether it is full or incremental) since the last backup and that would be based on the time here. I think, as you allude to, this sentence works just fine without "to a certain time" as scheduled backups almost explicitly apply this chain of backups, which is then referred to in the next sentence.